Dockerfile

```diff
@@ -66,4 +58,6 @@ LABEL maintainer="LINE Open Source <dl_oss_dev@linecorp.com>"
 COPY --from=downloader /downloads/kubectl /usr/local/bin/kubectl
 COPY --from=downloader /downloads/kustomize /usr/local/bin/kustomize
 
+RUN set -ex; kubectl && kustomize
```

Comment:
How about moving this to additional test stage?
(I prefer removing unnecessary layer from runtime image, but this is fine 🙂)